### PR TITLE
Output restructuring

### PR DESCRIPTION
**Pull request by LadyChristina**
All Submissions:

* [x] Have you followed the guidelines in our [Contributing documentation](https://blockchain-technology-lab.github.io/consensus-decentralization/contribute)?
* [x] Have you verified that there aren't any other open Pull Requests for the same update/change?
* [x] Does the Pull Request pass all tests?

# Description
Restructured output so that when there are multiple files generated for some purpose (e.g. mapped data) they go to a dedicated folder.
Also stopped generating (empty) mapped data files for timeframes where no data exists

### CODE REVIEW

File: tests/test_5_analyze.py [hunk 1]
```diff
@@ -18,22 +18,20 @@ def setup_and_cleanup():
...
-    }
+    csv_per_timeframes = {'2018': 'Entity Group,Entity,Resources\n'
+                                  '1AM2f...9pJUx/3G7y1...gPPWb,1AM2f...9pJUx/3G7y1...gPPWb,4\n'
+                                  'BTC.TOP,BTC.TOP,2\n'
+                                  'GBMiners,GBMiners,2\n'
+                                  'Unknown,1AM2fYfpY3ZeMeCKXmN66haoWxvB89pJUx,1',
+                          '2018-02': 'Entity Group,Entity,Resources\n'
+                                     '1AM2f...9pJUx/3G7y1...gPPWb,1AM2f...9pJUx/3G7y1...gPPWb,4\n'
+                                     'BTC.TOP,BTC.TOP,2\n'
+                                     'GBMiners,GBMiners,2', '2018-03': 'Entity Group,Entity,Resources\n'
```
Comment (dimkarakostas) [inline tests/test_5_analyze.py:29]: missing newline here

**Review by dimkarakostas — APPROVED** [review]
LGTM - fix the typo and merge

File: tests/test_5_analyze.py (hunk 1 above)
Comment (LadyChristina) [reply tests/test_5_analyze.py:29]: sorry I don't understand why we need a newline here, could you elaborate please?

Comment (LadyChristina) [reply tests/test_5_analyze.py]: ah right, I didn't see there was a different date there, nice catch - fixed now

**Pull request merged**